### ISSUE TITLE
The original 0.4.1 works

### ISSUE THREAD

**Issue by IBananaShake** [prof79/fansly-downloader-ng#82]
The original 0.4.1 works for downloading images, it just can't download m3u8 files

So whatever you did in your versions had made it not work.
So if you can just take the 0.4.1 version, and make it download m3u8 files, then it's perfect

**Comment by lucifa42** [comment]
0.8.17 still works for me for m3u8 files. I haven't upgraded since then and it's still trucking on.

**Comment by IBananaShake** [comment]
> 0.8.17 still works for me for m3u8 files. I haven't upgraded since then and it's still trucking on.

Can you still download that version?

**Comment by lucifa42** [comment]
Yep https://github.com/prof79/fansly-downloader-ng/releases/tag/v0.8.17

**Comment by waswosi** [comment]
Works perfectly for me. Exept it can't download comments. 
How reliable is this duplicate finder. Look here for example:

GRAND TOTAL DOWNLOAD SUMMARY
  Finished downloading media for 1 creators!
  Total timeline media: 1056 pictures & 694 videos (= 1750 items)
  Total message media: 0
  Total media (timeline & messages): 1750
  Downloaded media: 0 pictures & 1 videos (= 1 items)
  Duplicates skipped: 1756
  Missing items: 0

Can I assume that I have everything? Thank you.
